barretenberg/cpp/pil/avm/avm_mini.pil

```diff
@@ -108,4 +125,32 @@ namespace avmMini(256);
     // Same for the relations related to the error activation:
     // (ib * inv - 1 + op_div_err) = 0 && op_err * (1 - inv) = 0 
     // This works in combination with op_div_err * (sel_op_div - 1) = 0;
-    // Drawback is the need to paralllelize the latter.
\ No newline at end of file
+    // Drawback is the need to paralllelize the latter.
+
+
+    //===== CALL_RETURN ========================================================
+    // The program counter in the next row should be equal to the value loaded from the ia register
+    // This implies that a load from memory must occur at the same time
+    // Imply that we must load the jump selector into mema
+    #[return_pointer_increment]
+    sel_internal_call * ( internal_return_ptr' - ( internal_return_ptr + 1)) = 0; // we increment our jump selector
```

Comment:
@Maddiaa0 We might use another term than "jump selector" as the operations are not named JUMP anymore.